src/app/commands/scoap-squad/ScoapSquad.ts

```diff
@@ -0,0 +1,131 @@
+import {
+	ApplicationCommandPermissionType,
+	CommandContext,
+	CommandOptionType,
+	SlashCommand,
+	SlashCreator,
+} from 'slash-create';
+import roleIds from '../../service/constants/roleIds';
+import ServiceUtils from '../../utils/ServiceUtils';
+import CreateNewScoapPoll from '../../service/scoap-squad/CreateNewScoapPoll';
+import ValidationError from '../../errors/ValidationError';
+import discordServerIds from '../../service/constants/discordServerIds';
+
+module.exports = class ScoapSquad extends SlashCommand {
+	constructor(creator: SlashCreator) {
```

Comment:
I've seen `"tabWidth": 4,` in `.prettierrc` but I'm not seeing the formatter applied here. Did I miss something?

Reply:
We opted to not validate with .prettier and just use eslint via `yarn lint` and `yarn format`. I see this PR is out of date with develop so once it's in sync, the warnings should get thrown https://github.com/BanklessDAO/discord-bots/blob/develop/package.json

Reply:
What was the problem with `.prettier`? I'm just curious.

Reply:
No issues with .prettier specifically, just trying to reduce the number of dependencies and keep the project consistent. At minimum we should make sure the formatting style is the same for .prettier and eslint